README.md

```diff
@@ -1,70 +1,76 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> One paragraph statement about the project.
 
-In the project directory, you can run:
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Kindly Add a description of the project in the readme to make it more professional. Also, the link to the screenshot is broken so remember to fix it so that the screenshot can be displayed.